src/main/todos.ts

```diff
@@ -26,8 +42,26 @@ export function createTodo(todo: BasecampTodoRequest, todolistIdentifier: Todoli
  * @param todo payload to replace the existing todo
  * @param todoIdentifier id of the existing todo to replace
  */
-export function updateTodo(todo: BasecampTodoRequest, todoIdentifier: TodoIdentifier): void {
-    sendBasecampPutRequest(getUpdateTodoUrl(todoIdentifier), todo);
+export function updateTodo(request: BasecampTodoRequest | undefined, todoIdentifier: TodoIdentifier): void {
+
+    if(request == undefined) {
+        throw new BasecampRequestMissingError("Missing basecamp request!");
+    } else {
+        Logger.log(`Updating existing todo: "${request.content}"...\n`)
+        sendBasecampPutRequest(getUpdateTodoUrl(todoIdentifier), request);
+    }
+}
```

Comment:
Same comment as above for the `createTodo` function with the request being undefined - https://github.com/andrewc5716/onestop-basecamp-integration/pull/14/files#r1800223947